sdk/java-sdk/src/main/scala/com/akkaserverless/javasdk/impl/AnySupport.scala

```diff
@@ -195,7 +196,8 @@ class AnySupport(
     descriptors: Array[Descriptors.FileDescriptor],
     classLoader: ClassLoader,
     typeUrlPrefix: String = AnySupport.DefaultTypeUrlPrefix,
-    prefer: AnySupport.Prefer = AnySupport.Prefer.Java) {
+    prefer: AnySupport.Prefer = AnySupport.Prefer.Java,
+    serializer: Serializer) {
```

Comment:
Maybe `customSerializer` or something conveying that it is not usually used?

Reply:
In other places I called it `additionalSerializer`. I think I missed to rename here.